include/deal.II/matrix_free/portable_matrix_free.h

```diff
@@ -380,10 +380,10 @@ namespace Portable
      * locally owned degrees of freedom and the ghost elements correspond to the
      * (additional) locally relevant dofs.
      */
+    template <typename MemorySpace>
```

Comment:
It would be better to use a different name because `MemorySpace` will expand to `MemorySpace::Host` or `MemorySpace::Default`. This is confusing.